mmhuman3d/core/visualization/renderer/torch3d_renderer/meshes.py

```diff
@@ -0,0 +1,540 @@
+from typing import Iterable, List, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+from pytorch3d.renderer import TexturesUV, TexturesVertex
+from pytorch3d.renderer.mesh.textures import TexturesBase
+from pytorch3d.structures import Meshes, list_to_padded, padded_to_list
+
+from mmhuman3d.models import SMPL, SMPLX
+from mmhuman3d.utils.mesh_utils import \
+    join_meshes_as_batch as _join_meshes_as_batch
+from .builder import build_renderer
+from .textures import TexturesNearest
+from .utils import align_input_to_padded
+
+
+class ParametricMeshes(Meshes):
+    """Mesh structure for parametric body models, E.g., smpl, smplx, mano,
+    flame.
+
+    There are 3 ways to initialize the verts:
+        1): Pass the verts directly as verts_padded (N, V, 3) or verts_list
+        (list of (N, 3)).
+        2): Pass body_model and pose_params.
+        3): Pass meshes. Could be Meshes or ParametricMeshes.
+            Will use the verts from the meshes.
+    There are 3 ways to initialize the faces:
+        1): Pass the faces directly as faces_padded (N, F, 3) or faces_list
+        (list of (F, 3)).
+        2): Pass body_model and will use body_model.faces_tensor.
+        3): Pass meshes. Could be Meshes or ParametricMeshes.
+        Will use the faces from the meshes.
+    There are 4 ways to initialize the textures.
+        1): Pass the textures directly.
+        2): Pass the texture_images of shape (H, W, 3) for single person or
+        (_N_individual, H, W, 3) for multi-person. `body_model` should be
+        passed and should has `uv_renderer`.
+        3): Pass the vertex_color of shape (3) or (V, 3) or (N, V, 3).
+        4): Pass meshes. Could be Meshes or ParametricMeshes.
+        Will use the textures directly from the meshes.
+    """
+    # TODO: More model class to be added (FLAME, MANO)
+    model_classes = {'smpl': SMPL, 'smplx': SMPLX}
```

Comment:
If we use upper case `MODEL_CLASSES` here, it will be easier to be distinguished from normal value.

Reply:
ok